djangocms_snippet/forms.py

```diff
@@ -64,3 +65,26 @@ def save(self, **kwargs):
         if commit:
             snippet.save()
         return snippet
+
+
+class SnippetPluginForm(forms.ModelForm):
+
+    class Meta:
+        model = SnippetPtr
+        fields = ("cmsplugin_ptr", "snippet_grouper")
+
+    def __init__(self, *args, **kwargs):
+        """
+        Initialise the form with the add button enabled to allow adding a new snippet from the plugin form. To enable
+        this the get_related_url method on the widget is monkey patched to build a URL for the Snippet admin instead of
```

Comment:
```suggestion
        this the get_related_url method on the widget is overridden to build a URL for the Snippet admin instead of
```
A monkey patch is when you update the code at runtime, I think this would be more accurately described as overriding, since this is more just using inheritance.